onegov/election_day/formats/election/wabstic_proporz.py

```diff
@@ -549,6 +614,7 @@ def import_election_wabstic_proporz(
     if election.domain == 'region' and districts and election.distinct:
         if principal.has_districts:
             if len(districts) != 1:
+                # FIXME: better error messages
```

Comment:
Here it would make sense to also elaborate. Why can't you write a better error message at this time?

Reply:
At the moment I do not understand the difference between the two cases, so a better FIXME Message would be "Separate error msg for the two cases". I still lack the overview to write a good and valid error message.

Reply:
Alright, then I would just elaborate. It's okay to write long FIXME's and to really write down your current thoughts and what you don't get yet. Otherwise the FIXME will be very cryptic to yourself when you stumble upon it in the future :)

Reply:
I dont have the insigth yet in the swiss voting system to come up with a better error message at the time.